policies/rust/max_unsafe_blocks.py

```
@@ -0,0 +1,43 @@
+            c.skip("Unsafe block data not collected")
+
+        count_node = unsafe_blocks.get_node(".count")
+        count = count_node.get_value() if count_node.exists() else 0
+
+        if count > max_unsafe:
+            # Build location summary
+            loc_summary = ""
+            locations = unsafe_blocks.get_node(".locations")
+            if locations.exists() and locations.get_value():
+                locs = locations.get_value()[:5]
+                loc_strs = [f"{l.get('file', '?')}:{l.get('line', '?')}" for l in locs]
```

Ruff (0.15.1)

[error] Ambiguous variable name: `l`

(E741)